browser/src/control/Control.Mention.ts

```diff
@@ -80,10 +81,10 @@ class Mention extends L.Control.AutoCompletePopup {
 		if (this.filteredUsers.length !== 0) {
 			for (const i in this.filteredUsers) {
 				const entry = {
-					text: this.filteredUsers[i].username,
+					text: this.filteredUsers[i].label,
```

Comment:
Just from reading the code: Does it make sense to fallback to the username here in case the integrator does not send a label?

Reply:
Good point! I was thinking of that at the time, but seems I missed it actually